framework/app.mk

```diff
@@ -40,6 +40,11 @@ ifneq ($(SUFFIX),)
   app_LIB_SUFFIX := $(app_LIB_SUFFIX)_$(SUFFIX)
 endif
 
+ifeq ($(MOOSE_SKIP_DOCS),)
+  DOCUMENTATION := yes
+else
+  DOCUMENTATION := no
```

Comment:
I'd prefer not to introduce a "duplicate" variable. Can we just use the value of `$MOOSE_SKIP_DOCS` directly?

Reply:
seems like this should be fixed?
```
ifeq ($(MOOSE_SKIP_DOCS),)
```
ok with a new issue here too

Reply:
Fix `$MOOSE_SKIP_DOCS`? We can standardize on `TRUE` and `FALSE` instead of trying to handling things like `YES`, `NO`, etc. That would be preferable to setting more variables that do the same thing.